cdc/model/mounter.go

```diff
@@ -13,17 +13,19 @@
 
 package model
 
-// PolymorphicEvent describes an event can be in multiple states
+// PolymorphicEvent describes an event can be in multiple states.
 type PolymorphicEvent struct {
 	StartTs uint64
 	// Commit or resolved TS
 	CRTs uint64
+	// Identify whether the resolved event is in batch mode.
+	Batch bool
```

Comment:
I don't know if mode or batch is better?

Reply:
Also we probably need more documentation on the batch pattern in the code repository, maybe a design document, maybe comments.

Reply:
This is commented on the [`FlushRowChangedEvents `](https://github.com/pingcap/tiflow/pull/5342/files#diff-d9a3c1b05954cf385789af0a9c06058c18499bebb51676063237d144af0a61daR57) of sink

Reply:
> I don't know if mode or batch is better?
This field may be accessed directly from the outside. Ref: https://github.com/pingcap/tiflow/pull/5342/files#diff-176f4cdb8edddd8b6ef31640086f43f3b7d76d969f8a69db5686e87340a03e3dR304